### ISSUE TITLE
Printing ISO media size documents n-up onto non-ISO media sizes wastes space

### ISSUE THREAD

**Issue by michaelrsweet** [apple/cups#3560]
Version: -feature
CUPS.org User: twaugh.redhat

If an A4 PDF document is printed with number-up=2 onto a US Letter page, there is wasted space at the sides which it is possible to reduce.

This is because the A4 PDF document is first converted to PostScript in US Letter size (with pdftops or ghostscript), widening each page in the document.

If the document were converted to PostScript using the document's native page size (A4 in this case), it would be possible to lay out the document's pages at a slightly larger scale in the resulting output.

In other words, rather than increasing the width of each document page to match US Letter aspect ratio, it would be better to increase the height of each document page as it is laid out in the output.

Original bug report:
  https://bugzilla.redhat.com/show_bug.cgi?id=543183
